src/components/StealthCopiable/index.tsx

```diff
@@ -5,9 +5,9 @@ import useClipboard from 'react-use-clipboard'
 import recursivelyGetChildrenString from '../../helpers/recursivelyGetChildrenString'
 
 const CopyButton = styled.button`
-  background-color: ${({ theme }) => theme.colorsDeprecated.transparent};
+  background-color: transparent;
   border: none;
-  color: ${({ theme }) => theme.colorsDeprecated.primary};
+  color: ${({ theme }) => theme.colors.primary.text};
```

Comment:
```suggestion
  color: ${({ theme }) => theme.colors.primary.textWeak};
```

Weak context as background is transparent or light